crates/biome_js_analyze/src/lib.rs

```diff
@@ -204,7 +203,7 @@ mod tests {
             dependencies_index: Some(1),
             stable_result: StableHookResult::None,
         };
-        let rule_filter = RuleFilter::Rule("style", "useNodejsImportProtocol");
+        let rule_filter = RuleFilter::Rule("a11y", "noAriaHiddenOnFocusable");
```

Comment:
You'll need to revert the changes to the quick test.

Reply:
Already done! 👍